src/main/java/frc/robot/RobotContainer.java

```diff
@@ -92,14 +100,31 @@ public RobotContainer() {
      * passing it to a {@link edu.wpi.first.wpilibj2.command.button.JoystickButton}.
      */
     private void configureButtonBindings() {
+         /* 
         swerveSubsystem.setDefaultCommand(new RunCommand(() -> {
             double xPower = -driveController.getLeftY();
             double yPower = -driveController.getLeftX();
             double angularPower = -driveController.getRightX();
             swerveSubsystem.setSwerveDrivePowers(xPower, yPower, angularPower);
         }, swerveSubsystem));
+        */
     }
 
+
+    void periodic(){
```

Comment:
ideally we don't have to have a periodic method here, it would be best if it can be a defaultCommand like it was for swerve.

I wonder if it's possible to have both swerve and tank classes extend a common Drivetrain class. then we can just have an if statement depending on whether the current drivetrain extends Swerve or extends Tank, and set the default command of the drivetrain correspondingly